### ISSUE TITLE
How to add class,subclass or or static singleton after removal of create() in tinymce 6?

### ISSUE THREAD

**Issue by SwapnilDwivedi77** [tinymce/tinymce#7954]
I am upgrading from v5 to v6 and i have used `create()` to register static singleton. In the docs for tinymce 6 documentation i couldnt find an alternative to create. It only mentions it was deprecated in 5.10 and marked for removal in 6.

**Comment by TheSpyder** [comment]
This concept was designed for earlier versions of TinyMCE (before version 5) where extensibility was more common, and older browsers with limited JavaScript functionality were supported. In modern APIs and modern browsers there are better ways to achieve this.

Feel free to replicate the code if you feel you need it. It's still in the version 5 branch. The technique is fairly simple (but can be hard to follow due to the use of short variable names).
https://github.com/tinymce/tinymce/blob/05989b03bf56d98aaf3522630b481616799c4dc8/modules/tinymce/src/core/main/ts/api/util/Tools.ts#L174-L260
